torch/onnx/verification.py

```diff
@@ -158,7 +167,8 @@ def _compare_ort_pytorch_outputs(
             # TODO: Remove `check_shape` option once every shape inconsistent issue is addressed.
             if not check_shape:
                 # Allow different but broadcastable output shapes.
-                ort_out, pt_out = np.broadcast_arrays(ort_out, pt_out)
+                ort_out, pt_out_np = np.broadcast_arrays(ort_out, pt_out)
+                pt_out = torch.from_numpy(pt_out_np)
```

Comment:
Had to add this, or lintrunner would yell at me, as we expect pt_out as Sequence[Tensor], but np.broadcast_arrays makes it numpy